modules/macrel/contigs/main.nf

```diff
@@ -0,0 +1,38 @@
+process MACREL_CONTIGS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::macrel=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/macrel:1.1.0--py36hc5360cc_0':
+        'quay.io/biocontainers/macrel:1.1.0--py36hc5360cc_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*/*.smorfs.faa")     , emit: smorfs
+    tuple val(meta), path("*/*.all_orfs.faa")   , emit: all_orfs
```

Comment:
Would be good to compress everything in `.faa` after the main macrel command has finished :)

Reply:
Ok- all done now :)